hack/go-fmt.sh

```diff
@@ -9,7 +9,6 @@ else
     --env IS_CONTAINER=TRUE \
     --volume "${PWD}:/go/src/github.com/openshift/installer:z" \
     --workdir /go/src/github.com/openshift/installer \
-    --entrypoint sh \
-    quay.io/coreos/golang-testing \
+    openshift/origin-release:golang-1.10 \
```

Comment:
Can we keep the registry prefix?  What if a user is using something based on containers/image (e.g. podman) to pull these and they happen to have neither quay.io nor docker.io in their default search config (docs [here][1] and [here][2]).

[1]: https://github.com/containers/image/blob/4d1b61cb32c6aa82a0ad2bad43350a9f5c3bc0a2/docs/containers-registries.d.md#registries-scopes-and-search-order
[2]: https://github.com/containers/image/blob/4d1b61cb32c6aa82a0ad2bad43350a9f5c3bc0a2/docs/containers-registries.conf.5.md#example

Reply:
For some strange reason this image isn't present on quay (CC: @smarterclayton @stevekuznetsov).

Reply:
It's not a released image for use by customers, so we do not mirror it to quay.

Reply:
> It's not a released image for use by customers, so we do not mirror it to quay.

So should we have a `docker.io/` prefix here?

Reply:
SGTM